plugins/modules/nfs.py

```diff
@@ -196,9 +197,6 @@
             choices: [absent, present]
             default: present
 
-notes:
```

Comment:
Instead of removing this, better like this
**The I(check_mode) is supported.**

Reply:
done